docs/docs/gatsby-on-linux.md

```diff
@@ -64,10 +62,11 @@ Or to install all at the same time and approve `(y)` all installs:
 sudo apt update && sudo apt -y upgrade && sudo apt install build-essential && sudo apt install git && sudo apt install libpng-dev
```

Comment:
Nice work, can you write all commands separately with the proper description, so that users know about what the command will do on execution.

Reply:
Sure, and another thing I was wondering is, I have installed gatsby a couple times on both windows and linux, and never had to install build-essential (on linux) or windows-build-tools (on windows), but in the docs it says you have to install them, are they really necessary?

Reply:
I think you do need them for gatsby-plugin-sharp and probably some other plugins. I'm unsure if gatsby core needs it.

Reply:
`sharp` ships with binaries for windows, no build tools required.

Reply:
http://sharp.pixelplumbing.com/en/stable/install/

Reply:
Sharp tries to use supplied binaries first but will sometimes fall back to compiling source. http://sharp.pixelplumbing.com/en/stable/install/

Reply:
Marking as resolved since @laurieontech fixed this.